src/core/tsi/local_transport_security.h

```diff
@@ -33,7 +33,7 @@
  * Main struct for local TSI handshaker. All APIs in the header are
  * thread-comptabile.
  */
-typedef struct local_tsi_handshaker local_tsi_handshaker;
+typedef struct grpc_local_tsi_handshaker grpc_local_tsi_handshaker;
```

Comment:
nit: it does not seems we need typedef here. local_tsi_handshaker is not used anywhere else except local_tsi_handshaker.cc. We can remove this typedef.

If needed, this is in TSI namespace, use tsi_local_tsi_handshaker rather than grpc_local_tsi_handshaker and also change .cc accordingly.